screens/SendSetupScreen.js

```diff
@@ -144,29 +151,37 @@ const ErrorContainer = styled(View)`
   background-color: ${props => props.theme.danger700};
 `;
 
-const parseQr = (qrData: string): { address: string, amount: ?string } => {
-  let address = null;
-  let amount = null;
-
-  // Parse out address and any BIP21 relevant data
-  const parts = qrData.split("?");
-
-  address = parts[0];
-  const parameters = parts[1];
-  if (parameters) {
-    const parameterParts = parameters.split("&");
-    parameterParts.map(param => {
-      const [name, value] = param.split("=");
-      if (name === "amount") {
-        amount = value;
-      }
-    });
-  }
-  return {
-    address,
-    amount
-  };
-};
+// Only allow numbers and a single . in amount input
+const formatAmountInput = (amount: string, maxDecimals: number): string => {
```

Comment:
Remove from here, this function exists in
`utils/balance-utils`, re-use the existing one if possible

Reply:
removing